packages/salesforcedx-vscode-core/test/vscode-integration/orgPicker/orgList.test.ts

```diff
@@ -245,7 +243,7 @@ describe('orgList Tests', () => {
         expect(authList[0]).to.equal('alias1 - test-username1@example.com');
       });
 
-      it('should flag the org as expired if expiration date has passed', async () => {
+      it('should filter the list to ignore expired orgs', async () => {
```

Comment:
Why updating vscode-integration test? I think the test has been deprecated and there is no plan to use it again. I would suggest a jest unit test for the change you made in `orgList.ts`.

Reply:
I know we're not running them in GHA, but I try to keep them up to date anyways because they can still be run locally and provide some direction.